Fix homepage address and add landing page with image slider

jshortz opened the pull request.

jshortz committed:
Added exact before component to get router to properly route / path

jshortz committed:
Added slider components. Made separate CSS file for slider and removed slider CSS from App.css

jshortz committed:
Added Slider component to LandingPage.js

jshortz committed:
Added next and previous image methods to sliders and arrows

jshortz committed:
basic functionality of slider added. Arrows not displaying correctly.

jshortz committed:
Arrows changed to gt/lt, everything working. Still using default images, but component works.

pratima-shegokar reviewed (commented):
Gone through  PR and I have request regarding slider can you please increase the size of it and put little background to it also remove extra blanck lines from code which is added accidently.

jshortz committed:
added componentdidmount to change bg color, changed width in Slider.css to make bigger

jshortz committed:
Removed blank line from Slide.js

jshortz commented:
> Gone through PR and I have request regarding slider can you please increase the size of it and put little background to it also remove extra blanck lines from code which is added accidently.

Increased size as much as possible without messing up slider. Changed background to same purple as other pages (likely to change in future). Removed blank lines found,.

jshortz committed:
Merge branch 'master' into FE-fix-homepage-address

jshortz committed:
Update LandingPage.js

Added new background to LandingPage

jshortz committed:
Merge branch 'master' into FE-fix-homepage-address

jshortz committed:
Resolved conflicts in App.js. Again.

pratima-shegokar reviewed (approved):
checked .Resolve conflicts .Look good to me.
